### PR TITLE
Add prettier

### PR DESCRIPTION
**Pull request by anbraten**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "semi": true,
+  "trailingComma": "all",
+  "singleQuote": true,
```
Comment (pat-s) [inline .prettierrc:4]: Why single quotes? (just curious)

**Review by pat-s — COMMENTED** [review]
Big fan of this! 

We could in addition add a `pre-commit` CI check to make sure all new commits follow the formatting rules?

File: .woodpecker/test.yml [hunk 2]
```diff
@@ -58,9 +57,9 @@ steps:
     image: *golang_image
     group: test
     commands:
-      - "make generate-swagger"
-      - "DIFF=$(git diff | head)"
-      - "[ -n \"$DIFF\" ] && { echo \"swagger not up to date, exec 'make generate-swagger' and commit\"; exit 1; } || true"
+      - 'make generate-swagger'
+      - 'DIFF=$(git diff | head)'
+      - '[ -n "$DIFF" ] && { echo "swagger not up to date, exec ''make generate-swagger'' and commit"; exit 1; } || true'
```
Comment (6543) [inline .woodpecker/test.yml:62]: does `''` escape `'` ?

File: .prettierrc (hunk 1 above)
Comment (anbraten) [reply .prettierrc:4]: Its seems to be quite common. Just copied it.

**Comment by 6543** [comment]
Please resolve conflict

**Review by 6543 — APPROVED** [review]

**Pull request merged**